pkg/system/phase1_verifying.go

```diff
@@ -252,3 +273,26 @@ func (r *Reconciler) checkExternalPg(postgresDbURL string) error {
 	}
 	return nil
 }
+
+func (r *Reconciler) checkBucketLoggingPVC() error {
```

Comment:
Add a short description about what this function does.

Reply:
Done.